composer.json

```diff
@@ -21,9 +21,10 @@
 		"nette/di": "~2.3@dev",
 		"nette/http": "~2.3@dev",
 
-		"kdyby/curl-ca-bundle": "~1.0",
+		"composer/ca-bundle": "~1.0",
 		"ext-curl": "*",
-		"ext-json": "*"
+		"ext-json": "*",
+		"symfony/process": "~3.1"
```

Comment:
Because of this suggest in composer/ca-bundle: https://github.com/composer/ca-bundle/blob/master/composer.json#L34